files/image_config/syspoll/syspoll

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import json
+import time
+
+progname = sys.argv[0]
+
+datadir = '/var/platform'
+if not os.path.isdir(datadir):
+    if os.path.exists(datadir):
+        os.remove(datadir)
+    os.mkdir(datadir)
+outfile = datadir + '/system'
+tmpfile = datadir + '/system.tmp'
+
+ticks_per_sec = os.sysconf(os.sysconf_names['SC_CLK_TCK'])
+
+while True:
+    d = {}
+    d['hostname'] = subprocess.check_output(['hostname']).rstrip()
```

Comment:
1. Why do we need to update hostname each second?
2. It would be simpler to open '/etc/hostname' instead of forking and running bash to run hostname